packages/pouch-storage/src/PouchStorageTable.ts

```diff
@@ -34,7 +34,20 @@ export type PouchDBSort = Array<
   string | { [propName: string]: 'asc' | 'desc' }
 >;
 
-function makePouchFilter(type: string, value: FilterValue | FilterValue[]) {
+type PouchFilter = OnlyOneProp<{
+  $eq: FilterValue | FilterValue[];
+  $neq: FilterValue | FilterValue[];
```

Comment:
Wish we could use the PouchDB types here instead - it looks like there's actually a bug here. `$neq` should be `$ne`. Obviously was a bug before this change, but if we used PouchDB's types it would pick that up? https://www.npmjs.com/package/pouchdb-find

In any case, can mark it with a follow up ticket or as a separate smaller PR as this would be a runtime change.

Reply:
The types for `$regex` also differ. Pouch types say it's a string, but we are using Regex